<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopManagerForm</class>
 <widget class="QWidget" name="ShopManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string>SID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ClientName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ProductName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total</string>
       </property>
      </column>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="inputPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>352</width>
         <height>391</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Input</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="sidLabel">
            <property name="text">
             <string>S&amp;ID</string>
            </property>
            <property name="buddy">
             <cstring>sidLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="sidLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="clientnameLabel">
            <property name="text">
             <string>&amp;ClientName</string>
            </property>
            <property name="buddy">
             <cstring>ClientnameComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="ClientnameComboBox"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="ProductNameLabel">
            <property name="text">
             <string>&amp;ProductName</string>
            </property>
            <property name="buddy">
             <cstring>ProductnameComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="ProductnameComboBox"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="quantityLabel">
            <property name="text">
             <string>&amp;Quantity</string>
            </property>
            <property name="buddy">
             <cstring>quantityLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="quantityLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="PriceLabel">
            <property name="text">
             <string>&amp;Price</string>
            </property>
            <property name="buddy">
             <cstring>priceLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="totalLabel">
            <property name="text">
             <string>&amp;Total</string>
            </property>
            <property name="buddy">
             <cstring>totalLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="totalLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="priceLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTreeWidget" name="clienttreeWidget">
            <column>
             <property name="text">
              <string>ClinetName</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>PhoneNumber</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Address</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QTreeWidget" name="producttreeWidget">
            <column>
             <property name="text">
              <string>ProductName</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Price</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Stock</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addPushButton">
            <property name="text">
             <string>&amp;Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="modifyPushButton">
            <property name="text">
             <string>&amp;Modify</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>352</width>
         <height>391</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTreeWidget" name="searchTreeWidget">
            <column>
             <property name="text">
              <string>SID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>ClientName</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>ProductName</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Price</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Quantity</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QComboBox" name="searchComboBox">
              <item>
               <property name="text">
                <string>ID</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ClientName</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ProductName</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Price</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Quantity</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="searchLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="searchPushButton">
            <property name="text">
             <string>&amp;Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
